.travis.yml

```diff
@@ -13,32 +12,20 @@ cache:
 
 matrix:
   include:
-    # Main tests on supported Python versions.
-    - python: "2.7"
-      env: TOXENV=py27
-    - python: "3.4"
-      env: TOXENV=py34
-    - python: "3.5"
-      env: TOXENV=py35
-    - python: "3.6"
-      env: TOXENV=py36
-    - python: "pypy-5.3.1"
-      env: TOXENV=pypy
-
-    # Linting, docs, and other non-test stuff.
-    - python: "3.4"
-      env: TOXENV=lint
-    - python: "3.5"
-      env: TOXENV=docs
-    - python: "2.7"
-      env: TOXENV=packaging
-
-    # Test we haven't broken our major dependencies.
-    - python: "2.7"
-      env: TOXENV=py27-twistedMaster
-
+    - python: 2.7
+      dist: trusty
+    - python: 3.4
+      dist: trusty
+    - python: 3.5
+      dist: trusty
+    - python: 3.6
+      dist: trusty
+    - python: 3.7
+      dist: xenial
+    - python: pypy-5.3.1
+      dist: trusty
 install:
-  - "pip install -U pip setuptools"
-  - "pip install -U tox"
+  - pip install -U pip setuptools
+  - pip install -U tox-travis
```

Comment:
I'm aware of this integration - but for some reason I don't remember I always avoided it and used the more explicit (verbose) system that you now removed...

If we switch over here, we should also switch it for wsproto (and possible other projects).
hyperlink uses the explicit form. 
hyperframe uses tox-travis.
h11 doesn't use tox (we probably should...)
mitmproxy uses the explicit form.

Reply:
Ah, I didn't realise there were reasons to avoid `tox-travis`. I'm happy to switch them and go around updating the CI setup whilst I'm at it. hyperframe is because I changed it recently, I'm happy to switch to whichever but prefer `tox-travis` as less CI stuff to figure out.

Reply:
Yes - I don't remember the reason - I just remember avoiding it... ¯\_(ツ)_/¯
Lets use tox-travis everywhere if we can - and see if it gives us troubles.